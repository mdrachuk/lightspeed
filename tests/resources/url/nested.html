<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Title</title>
</head>
<body>
<a href="{{ 'test/url' | url }}">one</a>
<a href="{{ '/test/url' | url }}">two</a>
</body>
</html>