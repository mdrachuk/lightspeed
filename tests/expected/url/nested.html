<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Title</title>
</head>
<body>
<a href="http://example.org/subsite/test/url">one</a>
<a href="http://example.org/subsite/test/url">two</a>
</body>
</html>